packages/cra-template-safe-app/template/src/App.tsx

```diff
@@ -15,29 +15,33 @@ const Container = styled.form`
 `;
 
 const App: React.FC = () => {
-  const safe = useSafe();
+  const { sdk, safe } = useSafeAppsSDK();
   const [submitting, setSubmitting] = useState(false);
+
   const submitTx = useCallback(async () => {
     setSubmitting(true);
     try {
-      const safeTxHash = await safe.sendTransactions([
-        {
-          to: safe.info.safeAddress,
-          value: '0',
-          data: '0x',
-        },
-      ]);
+      const { safeTxHash } = await sdk.txs.send({
+        txs: [
+          {
+            to: safe.safeAddress,
+            value: '0',
+            data: '0x',
+          },
+        ],
+      });
       console.log({ safeTxHash });
-      const safeTx = await safe.loadSafeTransaction(safeTxHash);
+      const safeTx = await sdk.txs.getBySafeTxHash(safeTxHash);
       console.log({ safeTx });
     } catch (e) {
       console.error(e);
     }
     setSubmitting(false);
-  }, [safe]);
+  }, [safe, sdk]);
+
   return (
     <Container>
-      <Title size="md">{safe.info.safeAddress}</Title>
+      <Title size="md">{safe?.safeAddress}</Title>
```

Comment:
shouldn't `safe` be always set? (looking at https://github.com/gnosis/safe-apps-sdk/pull/68/files#diff-301eef080af00a69a696e6909cdebd171073aae7d3eae63afaadb7682acbdfbcR5)